src/Adapter/Presenter/Product/ProductLazyArray.php

```diff
@@ -93,7 +104,9 @@ public function __construct(
         Link $link,
         PriceFormatter $priceFormatter,
         ProductColorsRetriever $productColorsRetriever,
-        TranslatorInterface $translator
+        TranslatorInterface $translator,
+        HookManager $hookManager = null,
```

Comment:
If you intend to inject these dependencies in the `ProductLazyArray` then it should be handled on its construction in `ProductPresenter` as well
And maybe beyond that in `ProductPresenterFactory`

Reply:
Thanks. I fixed.